modules/features/settings/src/main/java/au/com/shiftyjelly/pocketcasts/settings/whatsnew/WhatsNewPage.kt

```diff
@@ -191,6 +185,34 @@ private fun WhatsNewSubscriptionBadge(tier: UserTier) = when (tier) {
     UserTier.Free -> Box {}
 }
 
+@Composable
+private fun getButtonTitle(
+    state: UiState.Loaded,
+): String = when (state.feature) {
+    WhatsNewFeature.AutoPlay -> stringResource(state.feature.confirmButtonTitle)
```

Comment:
Is the thinking that we'll need to leave AutoPlay until the bookmarks feature is out of early access, but that at that point we could remove any code for the AutoPlay what's new dialog?

Reply:
That's correct.